frappe/public/js/frappe/form/sidebar/form_sidebar.js

```diff
@@ -71,6 +71,7 @@ frappe.ui.form.Sidebar = Class.extend({
 			this.sidebar.toggle(false);
 		} else {
 			this.sidebar.toggle(true);
+			$(document.body).toggleClass('no-sidebar', false);
```

Comment:
Doesn't feel like a clean solution, I think you should use `no-list-sidebar` flag which should only affect the sidebar in listview

Reply:
- When List sidebar is hidden `no-sidebar` is attached to hide it.
- The problem is when Form is loaded, `no-sidebar` class is not removed which results in it being hidden in FormView as well. 
- The only solution is to toggle `no-sidebar` to false

Reply:
No, I am saying you should use a `no-list-sidebar` class and it should hide only the list sidebar even if you are in form.